### PR TITLE
Fix image orientation issue

### PR DESCRIPTION
**Pull request by YayunHuang**
Ref: MUP-144
some images have `orientation` property, which the browser knows and can automatically rotate the image, but the property is lost when we mockup the image, so need to fix it manually.

IMG_2805-portrait

### CODE REVIEW

File: mockup_package/mockup/image_generator.py [hunk 1]
```diff
@@ -38,6 +38,21 @@ def __init__(self, original_img_path, phone_slug, device_info):
     def create_fit_resolution_image(self):
         device_ratio = self.d_size[0] / self.d_size[1]
         image = Image.open(self.original_img_path).convert("RGBA")
+
+        # Solve image orientation issue # noqa: E501
```
Comment (pkong-ds) [inline mockup_package/mockup/image_generator.py]: this line also need `noqa: E501`?

File: mockup_package/mockup/image_generator.py [hunk 2]
```diff
@@ -38,6 +38,21 @@ def __init__(self, original_img_path, phone_slug, device_info):
...
+        # ref: https://stackoverflow.com/questions/56854176/is-there-a-way-i-can-detect-the-image-orientation-and-rotate-the-image-to-the-ri # noqa: E501
+        for orientation in ExifTags.TAGS.keys():
+            if ExifTags.TAGS[orientation] == "Orientation":
+                break
```
Comment (pkong-ds) [inline mockup_package/mockup/image_generator.py]: I read the stackoverflow thread and did not find any answers implementing your logic here. Can you 

1. point the link to the SO answer you adopted?
2. maybe add some more context as python comments? like
    - what was the problem?
    - what was the fix?

Thanks!

**Review by pkong-ds — CHANGES_REQUESTED** [review]

File: mockup_package/mockup/image_generator.py (hunk 2 above)
Comment (YayunHuang) [reply mockup_package/mockup/image_generator.py]: @pkong-ds sor I put incorrect link, should be this one: https://stackoverflow.com/questions/13872331/rotating-an-image-with-orientation-specified-in-exif-using-python-without-pil-in
updated and added more context

**Comment by YayunHuang** [comment]
@pkong-ds rebased to main

**Review by pkong-ds — APPROVED** [review]
LGTM, will merge after rebase

**Comment by YayunHuang** [comment]
@pkong-ds rebased

**Pull request merged**